sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -14269,4 +14269,36 @@ public void testFilterWithNVLAndNotIn()
         )
     );
   }
+
+  @Test
+  public void testLatestBy() {
```

Comment:
test name is a bit too generic

Reply:
totally agree - renamed it to `testLatestByWithoutMaxBytes` at first but decided to go with `testLatestByOnStringColumnWithoutMaxBytesSpecified` to be more specific